spk/gitea/src/service-setup.sh

```diff
@@ -5,39 +5,42 @@ PATH="/var/packages/git/target/bin:${PATH}"
 if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 7 ]; then
     SYNOPKG_PKGHOME="${SYNOPKG_PKGDEST}"
 fi
+
+if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 6 ]; then
+    SYNOPKG_PKGHOME="${SYNOPKG_PKGVAR}"
+fi
```

Comment:
`-lt`  means _less than_, but it should be `-lt 7`.

So this block is obsolete, just update the exiting block above.